src/components/QueryCondition.vue

```diff
@@ -41,7 +41,6 @@
 				<ReferenceRelationDropDown
 					class="query-condition__references"
 					v-model="selectedReferenceRelation"
-					:disabled="true"
```

Comment:
This doesn't actually work yet, right? In the sense of producing the correct SPARQL? Then I would say let's keep it disabled for now until the user can actually use it.

Reply:
Yeah, I enabled it to test it but forgot to disable it back again.